plugins/inputs/redis_sentinel/redis_sentinel.go

```diff
@@ -0,0 +1,609 @@
+package redis_sentinel
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"net/url"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/go-redis/redis"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type RedisSentinel struct {
+	Servers  []string
+	Password string
```

Comment:
Please add the TOML tags here, i.e.
```suggestion
	Servers  []string `toml:"servers"`
	Password string `toml:"password"`
```
to make clear those fields are configured by the user.

Reply:
@srebhan Addressed in https://github.com/influxdata/telegraf/pull/10042/commits/5f82283cc4818494af1401a5caeee1dba65e4c9b